src/IndexSetAnalysis.h

```diff
@@ -8,31 +8,98 @@
 
 /***************************************************************************
  *
- * @file AutoIndex.h
+ * @file IndexSetAnalysis.h
  *
- * Defines classes to automatically compute the minimal indexes for a table
+ * Computes indexes for relations in a translation unit
  *
  ***************************************************************************/
 
 #pragma once
 
-#include "MaxMatching.h"
+#include "RamAnalysis.h"
+#include "RamRelation.h"
 #include "RamTypes.h"
+#include "Util.h"
 
+#include <cstring>
```

Comment:
fairly sure cstring, limits, and sstream aren't used and can be removed

Reply:
removed.